integration/test-integration/src/test/java/org/wso2am/micro/gw/tests/testCaseBefore/APIManagerInitializeTestCase.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2am.micro.gw.tests.testCaseBefore;
+
+import org.awaitility.Awaitility;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.wso2.am.integration.test.ClientAuthenticator;
+import org.wso2.am.integration.test.impl.RestAPIAdminImpl;
+import org.wso2.am.integration.test.impl.RestAPIPublisherImpl;
+import org.wso2.am.integration.test.impl.RestAPIStoreImpl;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
+import org.wso2.am.integration.test.utils.bean.DCRParamRequest;
+import org.wso2am.micro.gw.tests.apim.APIMLifecycleBaseTest;
+import org.wso2am.micro.gw.tests.context.MicroGWTestException;
+import org.wso2am.micro.gw.tests.util.HttpClientRequest;
+import org.wso2am.micro.gw.tests.util.HttpResponse;
+import org.wso2am.micro.gw.tests.util.Utils;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Objects;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+
+public class APIManagerInitializeTestCase extends APIMLifecycleBaseTest {
+
+    @BeforeTest(description = "start the mgw along with apim server")
+    void start() throws Exception {
+        URL configToml = getClass().getClassLoader().getResource("apim/config.toml");
```

Comment:
Let's use path.separater instread of direclty using  /

Reply:
`getClass().getClassLoader().getResource()` - this method is a platform independent. if you see the javadoc of the getResource() method, it clearly mentions that the path should be separated by "/". So, why use the path.separater?

Reply:
Just learnt what the getResource() method is about. Thanks. +1